tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -2133,7 +2131,11 @@ def test_malform_export_config(client, context):
                              'readIops': 1000,
                              'writeIops': 2000,
                          },
-                     }}
+                     },
+                     "cgroup": "mygroup",
```

Comment:
You're going to remove cgroup from the cattle API, so I assume this will go away.